core/src/main/java/org/elasticsearch/discovery/zen/ping/unicast/UnicastZenPing.java

```diff
@@ -64,7 +64,9 @@
 
     public static final String ACTION_NAME = "internal:discovery/zen/unicast";
 
-    public static final int LIMIT_PORTS_COUNT = 1;
+    public static final int LIMIT_FOREIGN_PORTS_COUNT = 1;
```

Comment:
Note that in AWS context, we might have very big nodes 128gb RAM with 1 master node 2 data nodes on the same machine.
It could happen that the master is started after so it gets 9302 port allocated.
I wonder if we should not increase this to 3... May be not relevant thought here.